modules/candidate_list/test/TestPlan.md

```diff
@@ -32,4 +32,4 @@ Incorrect PSCID/DCCID combinations in the filter form should not give such an er
 It should return that no results were found.
 16. Enter correct PSCID/DCCID combination and ensure that it loads correct timepoint_list page
 17. Remove access_all_profiles permission and ensure that PSCID links are still clickable.
-18. Change useEDC and useProjects config variables to false and ensure filters are removed from menu.
+18. Change useEDC and useProjects config variables to _no_ and ensure filters are removed from menu.
```

Comment:
The value in the table is "false". Maybe we should explicitely say to go in the Config Module, under Study configs to change the value to "no"

Reply:
I think so that would reinforce the testing of the configuration module. otherwise the person testing configuration would have to check each and every module in loris

Reply:
If the value in the table is "false", it should say "false". The backend is the single source of truth, the frontend can say whatever it wants and changes over time.

Reply:
Ignore my comment, I thought this was a spec, not a test plan when I wrote it.

For a test, I agree that the front end is better to use.